<h1> {{ title }} </h1> 
<p> {{ description }} </p>

## Education

Computational Biology PhD from the University of Kansas

Undergraduate: Math & Physics at Drury University

## Skills 

### Computational / Quantitative Skills

• Transcriptomic analysis and platform specific knowledge: bulk RNASeq, scRNASeq (10X), spatial RNASeq (NanoString GeoMx/CosMx) <br />
• Mathematical models: biochemical (deterministic and stochastic), agent-based, multiscale <br />
• Quantitative analysis of large datasets <br />
• Transcriptomics data pipelines: read alignment, differential expression analysis (Tuxedo suite and kallisto/sleuth) <br />
• Machine learning: classification, dimensionality reduction (PCA, MDS, UMAP, others), deep learning, adversarial learning, manifold learning, convolutional neural networks <br />
• Image Analytics: FIJI/ImageJ, Pillow (Python) <br />
• Docking: Protein/Protein (high- and low-resolution docking), template-based docking, partial structure/interface docking, small molecule screening, DNA-small molecule docking <br />
• Molecular Dynamics: Small molecule MD simulations using a variety of force fields <br />
• Statistical data analysis: linear and nonlinear regression, power analysis, multiple testing cor-rection, parametric and non-parametric tests, bootstrapping and other resampling/permutation methods, estimators (kernel density estimation, etc.), statistical inference, information theory and distribution divergence <br />
• Mathematical and physical theory: criticality, stochastic processes, dynamical systems, branching processes, graph theory <br />
• Quantitative modeling: Physical phenomena simulations (VPython) and spatial biochemical models (PyURDME) <br />
• Protein design and ensemble generation (PyRosetta) <br />
• Programming and scripting languages: Python (pandas, scikit-learn, numpy, scipy, tensor-flow), Mathematica, R, MATLAB, Perl, bash, C++, SQL, and FORTRAN <br />
• Markup languages and databases: XML, SBML, LaTeX, Markdown, YAML, HTML <br />
• High performance computing on clusters (SLURM, PBS and other queueing systems) <br />
• Version control and repositories: git, GitHub, BitBucket <br />
• Environment control: Docker, Anaconda <br />
• IDEs: vim, Jupyter, RStudio, Atom <br />
• Operating Systems: Windows, Mac, Linux (Ubuntu, Red Hat, CentOS, openSUSE) <br />

### Experimental Laboratory Skills
• RNA-Seq (NGS): sample preservation and preparation for the Illumina platform <br />
• Bacterial molecular genetics <br />
• Basic experimental microbiology: culture, plating, antibiotic selection, cell quantification <br />

### Other Domain-Specific Knowledge
• Immunology <br />
• Microbiology, biofilms, and microbial physiology <br />
• Population genetics and genomics <br />
• Population dynamics and epidemiological modeling <br />
• Monte Carlo methods <br />
• Hypothesis-driven research design <br />
• Scientific and technical writing <br />

## Teaching Experience

Bioinformatics : Lab Teaching Assistant. Fall 2018, Fall 2019, & Spring 2020.

Genetics : Discussion leader and primary grader. Summer 2017.

Introduction to Cellular and Molecular Biology : Lab Teaching Assistant and auxillary lecture grader. Fall 2016 & Fall 2017.

Introduction to Organismal Biology: Lab Teaching Assistant and auxillary lecture grader. Spring 2016.

Introduction to Principles of Biology: Lab Teaching Assistant. Fall 2015.

## Publications

Gregory J Boggy, **GW McElfresh**, Eisa Mahyari, Abigail B Ventura, Scott G Hansen, Louis J Picker, Benjamin N Bimber, BFF and cellhashR: analysis tools for accurate demultiplexing of cell hashing data, Bioinformatics, Volume 38, Issue 10, 15 May 2022, Pages 2791–2801, https://doi.org/10.1093/bioinformatics/btac213

Huijing Wang, **GW McElfresh**<sup>*</sup>, Nishantha Wijesuriya, Adam Podgorny, Andrew D. Hecht, J. Christian J. Ray. Signatures of antibiotic tolerance and persistence in response to divergent stresses
doi: https://doi.org/10.1101/689851

Huijing Wang, **GW McElfresh**<sup>*</sup>, Nishantha Wijesuriya, Adam Podgorny, Andrew D. Hecht, J. Christian J. Ray. Overlapping stimulons arising in response to divergent stresses in Escherichia coli
doi: https://doi.org/10.1101/2021.12.17.473201

Chakravarty, D, **McElfresh, GW**, Kundrotas, PJ, Vakser, IA. How to choose templates for modeling of protein complexes: Insights from benchmarking template‐based docking. Proteins. 2020; 1– 12. https://doi.org/10.1002/prot.25875

**GW McElfresh**<sup>*</sup>, Christos Deligkaris. A vibrational entropy term for DNA docking with AutoDock. 2018.*Computational Biology and Chemistry*. doi.org/10.1016/j.compbiolchem.2018.03.027

<sup>*</sup> denotes first/co-first authorship.

## Book Chapters
 **GW McElfresh**<sup>*</sup>, Christian Ray. Intergenerational Cellular Signal Transfer and Erasure. 2018. *The Interplay of Thermodynamics and Computation in Natural and Artificial Systems*. Currently in press.
 
 <sup>*</sup> denotes first authorship.

## Presentations

**GW McElfresh**, Kathleen Busman-Sahay, Michael Nekorchuk, Sebastian Benjamin, Gregory J Boggy, Shana Feltham,
Eisa Mahyari, Scott G Hansen, Louis J Picker, Jacob D. Estes, Benjamin N Bimber. Integration of Spatial and Single Cell Transcriptomics Identifies Novel Pathologically Relevant Markers in SIV- and Mycobacterium tuberculosis-infected Rhesus Macaques. 2022. Nonhuman Primate Models for AIDS. 

**GW McElfresh**, Gregory J Boggy, Eisa Mahyari, Michael Stutz, Abigail B Ventura, Scott G Hansen, Louis J Picker, Benjamin N Bimber. Single Cell Transcriptomic Profiling of Early Tuberculosis Infection and Granuloma Formation in Rhesus Macaque. 2021. Nonhuman Primate Models for AIDS. 

**GW McElfresh**, Brian Drawert, Christian Ray. Discrete timescales of effects that stress signals have on the cell cycle. 2019. q-bio conference.

**GW McElfresh**, Brian Drawert, Christian Ray. Reconstructing the Role of Inheritance in Stress Signaling. 2018. Modeling of Protein Interactions. 

**GW McElfresh**, Christian Ray. Reconstructing the Role of Inheritance: Growth Arrest and Stress Signaling. 2018. Biomedical Sciences Symposium.

**GW McElfresh**, Christian Ray. Gene Expression Profile of E. Coli B REL606 Entering a Novel Antibiotic-Tolerant Persister State. 2016. Conference on Modeling of Protein Interactions

**GW McElfresh**, Cristos Deligkaris. Inclusion of Empirical Entropic Contributions to Binding Free Energy of Ligand-DNA Systems During Docking. 2014. American Chemical Society Midwest Regional Meeting.

**GW McElfresh**, Sheng-You Huang, and Xiaoquin Zou. Simultanous docking against multiple targets: Application to selectivity studies of protein kinase inhibitors. 2014. University of Missouri Summer Undergraduate Research Symposium.
