<h1> {{ title }} </h1> ![](https://scontent-msp1-1.cdninstagram.com/v/t51.2885-19/s150x150/90043375_209499150402264_225180608433750016_n.jpg?_nc_ht=scontent-msp1-1.cdninstagram.com&_nc_ohc=Y2O4jnBP0yMAX_MfgeR&oh=c4a858363459bc7dd51c3cb81b6f755e&oe=5EB17F7A)
<p> {{ description }} </p>

## Education

Current: Computational Biology PhD candidate at University of Kansas

Undergraduate: Math & Physics at Drury University

## Skills 

### Computational / Quantitative Skills

• Mathematical models: biochemical (deterministic and stochastic), agent-based, multiscale

• Quantitative analysis of large datasets

• Transcriptomics data pipelines: read alignment, differential expression analysis (Tuxedo suite and kallisto/sleuth)

• Machine learning: classification, dimensionality reduction (PCA, MDS, UMAP, others), deep learning, adversarial learning, manifold learning, convolutional neural networks

• Image Analytics: FIJI/ImageJ, Pillow (Python)

• Docking: Protein/Protein (high- and low-resolution docking), template-based docking, partial structure/interface docking, small molecule screening, DNA-small molecule docking

• Molecular Dynamics: Small molecule MD simulations using a variety of force fields

• Statistical data analysis: linear and nonlinear regression, power analysis, multiple testing cor-rection, parametric and non-parametric tests, bootstrapping and other resampling/permutation methods, estimators (kernel density estimation, etc.), statistical in-ference, information theory and distribution divergence

• Mathematical and physical theory: criticality, stochastic processes, dynamical systems, branching processes, graph theory

• Quantitative modeling: Physical phenomena simulations (VPython) and spatial biochemical models (PyURDME)

• Protein design and ensemble generation (PyRosetta)

• Programming and scripting languages: Python (pandas, scikit-learn, numpy, scipy, tensor-flow), Mathematica, R, MATLAB, Perl, bash, C++, SQL, and FORTRAN

• Markup languages and databases: XML, SBML, LaTeX, Markdown, YAML, HTML

• High performance computing on clusters (SLURM, PBS and other queueing systems)

• Version control and repositories: git, GitHub, BitBucket

• Environment control: Docker, Anaconda

• IDEs: vim, Jupyter, RStudio, Atom

• Operating Systems: Windows, Mac, Linux (Ubuntu, Red Hat, CentOS, openSUSE)

### Experimental Laboratory Skills
• RNA-Seq (NGS): sample preservation and preparation for the Illumina platform

• Bacterial molecular genetics

• Basic experimental microbiology: culture, plating, antibiotic selection, cell quantification


### Other Domain-Specific Knowledge
• Microbiology, biofilms, and microbial physiology

• Population genetics and genomics

• Population dynamics and epidemiological modeling

• Monte Carlo methods

• Hypothesis-driven research design

• Scientific and technical writing

## Teaching Experience

Bioinformatics : Lab Teaching Assistant. Fall 2018, Fall 2019, & Spring 2020.

Genetics : Discussion leader and primary grader. Summer 2017.

Introduction to Cellular and Molecular Biology : Lab Teaching Assistant and auxillary lecture grader. Fall 2016 & Fall 2017.

Introduction to Organismal Biology: Lab Teaching Assistant and auxillary lecture grader. Spring 2016.

Introduction to Principles of Biology: Lab Teaching Assistant. Fall 2015.

## Publications

Chakravarty, D, **McElfresh, GW**, Kundrotas, PJ, Vakser, IA. How to choose templates for modeling of protein complexes: Insights from benchmarking template‐based docking. Proteins. 2020; 1– 12. https://doi.org/10.1002/prot.25875

**GW McElfresh**, Christos Deligkaris. A vibrational entropy term for DNA docking with AutoDock. 2018.*Computational Biology and Chemistry*. doi.org/10.1016/j.compbiolchem.2018.03.027

## Book Chapters
 **GW McElfresh**, Christian Ray. Intergenerational Cellular Signal Transfer and Erasure. 2018. *The Interplay of Thermodynamics and Computation in Natural and Artificial Systems*. Currently in press.

## Presentations

**GW McElfresh**, Brian Drawert, Christian Ray. Discrete timescales of effects that stress signals have on the cell cycle. 2019. q-bio conference.

**GW McElfresh**, Brian Drawert, Christian Ray. Reconstructing the Role of Inheritance in Stress Signaling. 2018. Modeling of Protein Interactions. 

Huijing Wang<sup>*</sup>, **GW McElfresh**, Christian Ray. Insights into Bacterial Cellular Individuality with Regression on RNA-Seq Data. 2018. q-bio conference.

**GW McElfresh**<sup>*</sup>, Christian Ray. Reconstructing the Role of Inheritance: Growth Arrest and Stress Signaling. 2018. Biomedical Sciences Symposium.

**GW McElfresh**<sup>*</sup>, Christian Ray. Gene Expression Profile of E. Coli B REL606 Entering a Novel Antibiotic-Tolerant Persister State. 2016. Conference on Modeling of Protein Interactions

**GW McElfresh**<sup>*</sup>, Cristos Deligkaris. Inclusion of Empirical Entropic Contributions to Binding Free Energy of Ligand-DNA Systems During Docking. 2014. American Chemical Society Midwest Regional Meeting.

**GW McElfresh**<sup>*</sup>, Sheng-You Huang, and Xiaoquin Zou. Simultanous docking against multiple targets: Application to selectivity studies of protein kinase inhibitors. 2014. University of Missouri Summer Undergraduate Research Symposium.

<sup>*</sup> denotes presenting author.
